pkg/formulaexec/formula_exec.go

```diff
@@ -599,14 +602,14 @@ func execFormula(ws *workspace.Workspace, fc wfapi.FormulaAndContext, formulaCon
 		Codec:    0x71, // 0x71 means "dag-cbor" -- See the multicodecs table: https://github.com/multiformats/multicodec/
 		MhType:   0x20, // 0x20 means "sha2-384" -- See the multicodecs table: https://github.com/multiformats/multicodec/
 		MhLength: 48,   // sha2-384 hash has a 48-byte sum.
-	}}, nFormulaAndContext.(schema.TypedNode).Representation())
```

Comment:
I see a somewhat critical little variable naming typofix creeps in here as well... ;P